applications/nrf_desktop/README.rst

```diff
@@ -955,6 +955,42 @@ To use the nRF Desktop application with your custom board:
    Some options have dependencies that might not be needed when these options are disabled.
    For example, when the LEDs module is disabled, the PWM driver is not needed.
 
+Adding a new motion sensor
+==========================
+
+The nRF52 Desktop application comes with a motion module able to read data from a motion sensor.
+A communication with the sensor is done through a sensor driver API.
+In the scope of NCS two motion sensor drivers are available, for PMW3360 and PAW3212.
+It is likely that another sensor will be more suitable for a new board.
```

Comment:
Client will most probably use different sensor. It's not a doubt but the fact that other sensor that we don't have support for will most likely be used. But if it will be we cannot be 100% sure.
I will try to write it differently/

Reply:
I removed this sentence altogether.